### PR TITLE
feat:  Getting Started instructions for new KSD subscriber

### PR DESCRIPTION
**Pull request by Javlopez**

### CODE REVIEW

File: docs/trial/getting-started.md [hunk 1]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
 ---
 
 # KSD Trial - Getting Started
+Thank you for subscribe to Koor Storage Distribution.
 
-Coming soon...this section is under construction. Thanks for being patient while we set things up for you.
+Kook allows you to attach Ceph storage to Kubenetes clusters.
```
Comment (zalsader) [inline docs/trial/getting-started.md]:
```suggestion
Koor allows you to attach Ceph storage to Kubenetes clusters. 
```

File: docs/trial/getting-started.md [hunk 2]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
 ---
 
 # KSD Trial - Getting Started
+Thank you for subscribe to Koor Storage Distribution.
```
Comment (zalsader) [inline docs/trial/getting-started.md]:
```suggestion
Thank you for trying Koor Storage Distribution.
```

File: docs/trial/getting-started.md [hunk 3]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+└─ubuntu--vg-swap_1 swap              9492a3dc-ad75-47cd-9596-678e8cf17ff9   [SWAP]
+vdb
+```
+
+## Steps
+
+#### 1. Install Koor Storage Distribution Operator and Resources
+
+```bash
+$ git clone --single-branch --branch v1.11.0 https://github.com/koor-tech/koor.git
```
Comment (zalsader) [inline docs/trial/getting-started.md]: This works, but why don't we direct people to use helm?

File: docs/trial/getting-started.md [hunk 4]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
 ---
 
 # KSD Trial - Getting Started
+Thank you for subscribe to Koor Storage Distribution.
 
-Coming soon...this section is under construction. Thanks for being patient while we set things up for you.
+Kook allows you to attach Ceph storage to Kubenetes clusters. 
+Koor makes it easier to deploy and manage storage for use in Kubernetes. 
+On bare metal or virtual machines, in your own data center or one of the many cloud providers, 
+the Koor Storage Distribution (KSD) has the tools you need.
```
Comment (ideepika) [inline docs/trial/getting-started.md]:
```suggestion
the Koor Storage Distribution (KSD) has all that you need.
Configure any type of storage solution. Koor supports Block, Filesystem and S3-compatible Object Storage at one place.
```

File: docs/trial/getting-started.md [hunk 5]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+
+## Requirements
+
+Koor can be installed on any existing Kubernetes cluster
```
Comment (ideepika) [inline docs/trial/getting-started.md:17]:
```suggestion

## Kubernetes Prerequisites
Though KSD can be installed on any existing Kubernetes cluster. We support,
```

File: docs/trial/getting-started.md [hunk 6]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+
+- Kubernetes v1.21 or higher is supported
+- Architectures supported are `amd64` / `x86_64` and `arm64`.
+
+- Ceph Prerequisites: to configure the Ceph storage cluster, at least one of these local storage types is required
```
Comment (ideepika) [inline docs/trial/getting-started.md]:
```suggestion
## Ceph Prerequisites
To configure the Ceph storage cluster, at least one of these local storage types is required
```

File: docs/trial/getting-started.md [hunk 7]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+$ lsblk -f
+NAME                  FSTYPE      LABEL UUID                                   MOUNTPOINT
+vda
+└─vda1                LVM2_member       >eSO50t-GkUV-YKTH-WsGq-hNJY-eKNf-3i07IB
+├─ubuntu--vg-root   ext4              c2366f76-6e21-4f10-a8f3-6776212e2fe4   /
+└─ubuntu--vg-swap_1 swap              9492a3dc-ad75-47cd-9596-678e8cf17ff9   [SWAP]
+vdb
+```
+
+## Steps
```
Comment (ideepika) [inline docs/trial/getting-started.md]:
```suggestion
## Install Koor Storage Distribution Operator and Resources

### Using Koor fork
```

File: docs/trial/getting-started.md [hunk 8]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+rook-ceph-osd-0-6d8db96856-qw6ld        1/1     Running   0          1m
+rook-ceph-osd-1-5c76b5466-bncdt         1/1     Running   0          1m
+rook-ceph-osd-2-7fcf9745c5-t7vwf        1/1     Running   0          1m
+rook-ceph-tools-566fbd58cc-lfxqn        1/1     Running   0          1m
+```
+
+#### 3. Consume storage using Kubernetes Native APIs
+
+```bash
+kubectl create -f csi/rbd/storageclass.yaml
```
Comment (ideepika) [inline docs/trial/getting-started.md]:
```suggestion
$ kubectl create -f csi/rbd/storageclass.yaml
```

File: docs/trial/getting-started.md [hunk 9]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+kubectl create -f - << EOF
```
Comment (ideepika) [inline docs/trial/getting-started.md]:
```suggestion
$ kubectl create -f - << EOF
```

File: docs/trial/getting-started.md [hunk 10]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+spec:
+  accessMode:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: rook-ceph-block  
+EOF
+```
+
```
Comment (ideepika) [inline docs/trial/getting-started.md:127]: ### Using Helm Charts

#### 1. After deploying your Kubernetes cluster, install Koor Storage Distribution using [Koor Operator Helm chart](https://github.com/koor-tech/koor-operator/tree/feature/ksd-202#install-using-helm)

```
$ helm repo add koor-release https://charts.koor.tech/release
$ helm install --create-namespace koor-ceph koor-release/rook-ceph  # (optional) -f utils/operatorValues.yaml
```

#### 2. You should be able to see a ready-to-use Koor Storage Cluster

```
$ kubectl get -n koor-ceph pod
NAME                                   READY   STATUS    RESTARTS   AGE
rook-ceph-mgr-a-c5gff977d-656mt         1/1     Running   0          1m
rook-ceph-mgr-a-7f6bf57db6-nlprb        1/1     Running   0          1m
rook-ceph-mgr-b-85b77d9f8f-nd724        1/1     Running   0          1m
rook-ceph-mgr-c-fc6888db9-mswkq         1/1     Running   0          1m
rook-ceph-operator-6f4fdc4448           1/1     Running   0          2m
rook-ceph-osd-0-6d8db96856-qw6ld        1/1     Running   0          1m
rook-ceph-osd-1-5c76b5466-bncdt         1/1     Running   0          1m
rook-ceph-osd-2-7fcf9745c5-t7vwf        1/1     Running   0          1m
rook-ceph-tools-566fbd58cc-lfxqn        1/1     Running   0          1m
```

#### 3. You can then test basic Ceph Block storage 

```
$ kubectl apply -f https://github.com/rook/rook/blob/master/deploy/examples/csi/rbd/storageclass.yaml
$ kubectl create -f - << EOF
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: rbd-pvc
spec:
  accessMode:
    - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
  storageClassName: rook-ceph-block  
EOF
```

@zalsader please verify if this looks good, @Javlopez please adapt accordingly.

File: docs/trial/getting-started.md [hunk 11]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+That's it! :party_popper:, now you are using Koor Storage Distribution Operator.
+
+Do you have issues running Koor Operator?,
```
Comment (ideepika) [inline docs/trial/getting-started.md]: Maybe we should move it to Koor Knowledge Centre Troubleshooting page https://kb.koor.tech/knowledge/rook/issues/

File: docs/trial/getting-started.md (hunk 10 above)
Comment (zalsader) [reply docs/trial/getting-started.md:127]: LGTM

File: docs/trial/getting-started.md [hunk 12]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
 ---
 
 # KSD Trial - Getting Started
```
Comment (dave-at-koor) [inline docs/trial/getting-started.md]: Looks like we don't need this heading. The page title will show automatically.

File: docs/trial/getting-started.md [hunk 13]
```diff
@@ -3,5 +3,86 @@ KSD Trial - Getting Started
...
+      storage: 1Gi
+  storageClassName: rook-ceph-block  
+EOF
+```
+
+That's it! :party_popper:, now you are using Koor Storage Distribution Operator.
+
+Do you have issues running Koor Operator?, 
+review our blog [Koor Operator Troubleshooting](https://koor.tech/categories/troubleshooting/) 
+or [Contact us](https://koor.tech/contact/).
```
Comment (dave-at-koor) [inline docs/trial/getting-started.md]: Let's send people to the Help Desk page instead. [Contact us](/support/help-desk/)

**Review by dave-at-koor — CHANGES_REQUESTED** [review]
Looks great. Just a couple of changes from my POV.

**Review by dave-at-koor — APPROVED** [review]

**Pull request merged**